Library/Homebrew/tap.rb

```diff
@@ -536,15 +535,33 @@ def formula_renames
 
   # Hash with tap migrations
   def tap_migrations
-    require "json"
-
     @tap_migrations ||= if (migration_file = path/"tap_migrations.json").file?
```

Comment:
This could perhaps use a constant which is included in the `HOMEBREW_TAP_JSON_FILES` constant?